velox/common/encode/Base64.cpp

```diff
@@ -29,20 +31,23 @@ constexpr static int kBinaryBlockByteSize = 3;
 // Size of an encoded block in bytes (4 bytes = 24 bits)
 constexpr static int kEncodedBlockByteSize = 4;
 
-constexpr const Base64::Charset kBase64Charset = {
+// Character sets for Base64 and Base64 URL encoding
+constexpr const Charset kBase64Charset = {
     'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
     'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
     'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
     'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
     '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '+', '/'};
-constexpr const Base64::Charset kBase64UrlCharset = {
+
+constexpr const Charset kBase64UrlCharset = {
     'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
     'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
     'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
     'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
     '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '-', '_'};
 
-constexpr const Base64::ReverseIndex kBase64ReverseIndexTable = {
+// Reverse lookup tables for decoding
+constexpr const ReverseIndex kBase64ReverseIndexTable = {
```

Comment:
We should probably move this to the header so that we can reuse in the test.
It should be `static constexpr const ReverseIndex kBase64ReverseIndexTable =`